### ISSUE TITLE
taSelection load issue using oclazyload

### ISSUE THREAD

**Issue by garhbod** [fraywing/textAngular#942]
When using oclazyload to load textAngular  get the following error.

Unknown provider: taSelectionProvider

Similar to issue #608

**Comment by toxaq** [comment]
Yep, it seems one dependency error has been replaced with another.
taRegisterTool (which is in textAngularSetup) is dependent on taSelection (which is declared in textAngular).

**Comment by toxaq** [comment]
![ocloaded_textangular](https://cloud.githubusercontent.com/assets/1203104/11606442/b827e2ca-9b86-11e5-9dd8-525279e91adc.png)

Just for reference this is the error I see. I've tried all sorts of ways of putting the files together, declaring temporary providers etc but none of them have got me a working textAngular install. I'm trying to upgrade from 1.3.6 which used the manual solution from #866

**Comment by alo** [comment]
+1

**Comment by alo** [comment]
There's no problem with ocLazyLoad 0.5.0. Problems when updating ocLazyLoad to 1.0.9

**Comment by avinashjoshi** [comment]
+1

**Comment by jongunter** [comment]
+1 This issue. Please let us know if there's any way we could help getting this fixed.

**Comment by johansten** [comment]
@jongunter: Adding a dependency to textAngular.DOM in textAngularSetup gets me one step closer, I think. It still doesn't work, I get an undefined error in ocLazyload.

**Comment by johansten** [comment]
Of course, I had forgotten to lazy load rangy. Now it's working.

**Comment by johansten** [comment]
@jongunter, and everyone else:

Check https://github.com/ocombe/ocLazyLoad/issues/286 on how to get it to work,

**Comment by facuxt** [comment]
@johansten adding that line solved me the error... Thank you!!!!

https://github.com/johansten/textAngular/commit/26a90684638b93a0217386da22c1c77085dab0a4

i dont know why this is not included by default in textAngular..

**Comment by anasanzari** [comment]
+1
